test/immortal-child.ts

```diff
@@ -0,0 +1,70 @@
+import t from "tap";
```

Comment:
Separate out increasing test-coverage as a different PR, but only have tests for the code you've written for the PR.

Reply:
OK